index.html

```diff
@@ -2,10 +2,9 @@
   <head>
     <title>Sea of Awesomeness</title>
   </head>
-    <head>
-        <title>My awesome web page</title>
-    </head>
-
-    Hello there, awesome person!
+    <body>
+        <h1>My awesome web page</h1>
+    <img src="https://avatars0.githubusercontent.com/u/19331862?v=4
```

Comment:
You are missing a "> at the end of the <img> tag.

You have:
`<img src="https://avatars0.githubusercontent.com/u/19331862?v=4`

You need:
`<img src="https://avatars0.githubusercontent.com/u/19331862?v=4">`

Reply:
see the ending "> that is needed?

Reply:
I see it and I added it but nothing seems to be happening.